### PR TITLE
Spectroscopy faisst jan1624

### PR DESCRIPTION
**Pull request by afaisst**
Add the new use case spectroscopy notebook to the main.

Change includes a working version of the spectroscopy generator (only outline in the current main branch).

### CODE REVIEW

File: spectroscopy/spectra_generator.md [hunk 1]
```diff
@@ -76,22 +77,189 @@ Andreas Faisst, Jessica Krick, Shoubaneh Hemmati, Troy Raen, Brigitta Sipőcz, D
...
+```
+
+### 1.2 Write out your sample to disk
+
+At this point you may wish to write out your sample to disk and reuse that in future work sessions, instead of creating it from scratch again.
+
+For the format of the save file, we would suggest to choose from various formats that fully support astropy objects(eg., SkyCoord).  One example that works is Enhanced Character-Separated Values or ['ecsv'](https://docs.astropy.org/en/stable/io/ascii/ecsv.html)
+
+```python
+sample_table.write('data/input_sample.ecsv', format='ascii.ecsv', overwrite = True)
```
Comment (jkrick) [inline spectroscopy/spectra_generator.md:162]: This line gives me errors on the first run through, but on a second run it completes.  Now I can't reproduce, but the error implied to me that the 'data' directory doesn't exist before this line.

File: spectroscopy/spectra_generator.md [hunk 2]
```diff
@@ -76,22 +77,189 @@ Andreas Faisst, Jessica Krick, Shoubaneh Hemmati, Troy Raen, Brigitta Sipőcz, D
...
+
+```python
+%%time
+## Get DESI and BOSS spectra with SPARCL
+df_spec_DESIBOSS = DESIBOSS_get_spec(sample_table, search_radius_arcsec=5)
+df_spec.append(df_spec_DESIBOSS)
+```
+
+```python
+df_spec.data
```
Comment (jkrick) [inline spectroscopy/spectra_generator.md:248]: So I can show off the data structure a little more cleanly, I suggest:
```suggestion
df_spec.data.sort_values(by = ['objectid'])
```

**Review by jkrick — APPROVED** [review]
Two minor comments that will help me get the demo going.

File: spectroscopy/spectra_generator.md (hunk 1 above)
Comment (afaisst) [reply spectroscopy/spectra_generator.md:162]: yes, that's correct. "data/" needs to exist.

**Pull request merged**